pynamodb/models.py

```diff
@@ -357,6 +361,51 @@ def update_item(self, attribute, value=None, action=None, conditional_operator=N
                 setattr(self, name, attr.deserialize(value.get(ATTR_TYPE_MAP[attr.attr_type])))
         return data
 
+
+    def update(self, attributes,  conditional_operator=None, **expected_values):
+        """
+        Updates an item using the UpdateItem operation.
+
+        :param attributes: A dictionary of attributes to update in the following format
+                            {
+                                attr_name: {'value': 10, 'action': 'ADD'},
+                                next_attr: {'value': True, 'action': 'PUT'},
+                            }
+        """
+        if not isinstance(attributes, dict):
+            raise TypeError("the value of `attributes` is expected to be a dictionary")
+
+        self._conditional_operator_check(conditional_operator)
+        args, save_kwargs = self._get_save_args(null_check=False)
+        kwargs = {
+            pythonic(RETURN_VALUES):  ALL_NEW,
+            pythonic(ATTR_UPDATES): {},
+            'conditional_operator': conditional_operator,
+        }
+
+        if pythonic(RANGE_KEY) in save_kwargs:
+            kwargs[pythonic(RANGE_KEY)] = save_kwargs[pythonic(RANGE_KEY)]
+
+        if expected_values:
+            kwargs['expected'] = self._build_expected_values(expected_values, UPDATE_FILTER_OPERATOR_MAP)
+
+        for attr, params in attributes.items():
+            attribute_cls = self._get_attributes()[attr]
+            action = params['action'] and params['action'].upper()
```

Comment:
no, either `False` (in case `params['action']` is evaluated to `False`) or the value of the second statement)
>>> a = {'none': False, 'one': 'yes'}
```
>>> a['none'] and a['none'].upper()
False
>>> a['one'] and a['one'].upper()
'YES'
```